From akpm@linux-foundation.org  Fri Apr 18 14:55:35 2008
From: Adrian Bunk <bunk@kernel.org>
Date: Fri, 18 Apr 2008 13:53:56 -0700
Subject: PCI: pci_bus_size_cardbus() mustn't be __devinit
To: greg@kroah.com
Cc: linux-pci@atrey.karlin.mff.cuni.cz, akpm@linux-foundation.org, bunk@kernel.org
Message-ID: <200804182053.m3IKruWr015138@imap1.linux-foundation.org>


From: Adrian Bunk <bunk@kernel.org>

WARNING: drivers/pci/built-in.o(.text+0x28e1f): Section mismatch in reference from the function pci_bus_size_bridges() to the function .devinit.text:pci_bus_size_cardbus()

Signed-off-by: Adrian Bunk <bunk@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/pci/setup-bus.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/drivers/pci/setup-bus.c
+++ b/drivers/pci/setup-bus.c
@@ -407,8 +407,7 @@ static int pbus_size_mem(struct pci_bus 
 	return 1;
 }
 
-static void __devinit
-pci_bus_size_cardbus(struct pci_bus *bus)
+static void pci_bus_size_cardbus(struct pci_bus *bus)
 {
 	struct pci_dev *bridge = bus->self;
 	struct resource *b_res = &bridge->resource[PCI_BRIDGE_RESOURCES];
